docker/test/integration/minifi/__init__.py

```diff
@@ -93,12 +96,22 @@ def deploy_flow(self,
         if self.network is None:
             net_name = 'nifi-' + str(uuid.uuid4())
             logging.info('Creating network: %s', net_name)
-            self.network = self.client.networks.create(net_name)
+            # Set IP
+            ipam_pool = docker.types.IPAMPool(
+                subnet='192.168.42.0/24',
```

Comment:
Do you think it's going to work with whatever docker config developers may have?